app/modules/content-items/components/EditorBlock.js

```diff
@@ -59,7 +59,7 @@ const PureEditorBlock = (props: Props): React.Node => {
     <div>
       {contentItemDisplay}
       { subableContentItem && (
-        <div style={{ marginLeft: '2em' }}>
+        <div style={{ borderLeft: '0.2em solid lightgray', paddingLeft: '1rem' }}>
```

Comment:
Are there no utility classes in Semantic UI? Either way, when the LESS version is integrated this should be extracted into custom classes.

Reply:
Whoops, just noticed that's my code. Semantic UI doesn't have utility classes AFAIK because that goes against their philosophy (atomic CSS is bacisally as un-semantic as you can be). I temporarily put the inline styles there until I can create custom LESS code, like you said. Will be fixed in the feature-editable-content-items branch later on.